### ISSUE TITLE
issue running nvim after cloning your repo

### ISSUE THREAD

**Issue by edwinchenloo** [meuter/nvim#2]
On the first run I get:

`[mason-lspconfig.nvim] installing gopls`
`Failed to run `config` for indent-blankline.nvim`

`...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:369: module 'ibl' not found:`
`^Ino field package.preload['ibl']`
`cache_loader: module ibl not found`
`cache_loader_lib: module ibl not found`
`^Ino file './ibl.lua'`
`^Ino file '/__w/neovim/neovim/.deps/usr/share/luajit-2.1/ibl.lua'`
`^Ino file '/usr/local/share/lua/5.1/ibl.lua'`
`^Ino file '/usr/local/share/lua/5.1/ibl/init.lua'`
`^Ino file '/__w/neovim/neovim/.deps/usr/share/lua/5.1/ibl.lua'`
`^Ino file '/__w/neovim/neovim/.deps/usr/share/lua/5.1/ibl/init.lua'`
`^Ino file './ibl.so'`
`^Ino file '/usr/local/lib/lua/5.1/ibl.so'`
`^Ino file '/__w/neovim/neovim/.deps/usr/lib/lua/5.1/ibl.so'`
`^Ino file '/usr/local/lib/lua/5.1/loadall.so'`

`# stacktrace:`
`  - ~/.config/nvim/init.lua:176`
`Press ENTER or type command to continue`

It seems to be related to indent-blankline

**Comment by meuter** [comment]
Thanks for the report. It has been a while since I updated my plugins. I will try to clone the repo in a fresh VM to try and repro the problem.

**Comment by meuter** [comment]
I tried installing the config on a fresh Ubuntu 22.04 in WSL, and I cannot repro the problem. I tried a `Lazy update`  to update all packages and everything seems to work on my end; which I just commited for good measure.

Now googling the error message you reported, it seems that this might be related: 

https://github.com/lukas-reineke/indent-blankline.nvim/discussions/692

If you are encountering the same error as the OP in the issue above, it could be that you still have branch `v2` or indent_blankline lying around in your .local/share/nvim. And since my config relies on the `ibl` branch, Lazy could get mixed up. What I would suggest is to go into Lazy, uninstall indent-blankline completely, exit nvim and restart to trigger a re-install. Hopefully it should get rid of the error 🤞

Let me know how it goes.
